pkgs/test/test/util/fake_test.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore: deprecated_member_use
```

Comment:
Is this cruft? It shouldn't be necessary I don't think here

Reply:
You're right, this was a copy/paste error.  Fixed.